test/verify/check-realms

```diff
@@ -256,18 +256,29 @@ class TestRealms(MachineCase):
         # should have stopped cert tracking
         self.assertNotIn("status:", m.execute("ipa-getcert list"))
 
-        # Send a wrong password
-        b.click(self.domain_sel)
-        b.wait_popup("realms-op")
-        b.set_val(self.op_address, "cockpit.lan")
-        b.wait_attr(self.op_admin, "placeholder", 'e.g. "admin"')
-        b.set_val(self.op_admin, "admin")
-        b.set_val(self.op_admin_password, "foo")
-        b.click(".realms-op-apply")
-        b.wait_text_not(".realms-op-message", "")
-        b.wait_not_visible(".realms-op-leave-only-row")
-        b.click(".realms-op-cancel")
-        b.wait_popdown("realms-op")
+        # Sometimes with some versions of realmd the Leave operation
+        # from above is still active in the realmd daemon.  So we loop
+        # here until we get the expected error instead of "Already
+        # running another action".
+
+        while True:
```

Comment:
Right, that sounds like a bug, let's please at least report it in bz. Also, please let's not have an indefinite loop here, but at most one retry?

Reply:
> let's please at least report it in bz

It only seems to happen on the Debians, and I am not familiar with reporting bugs there... *hint* *hint*

Reply:
> Also, please let's not have an indefinite loop here, but at most one retry?

Done.